src/doc/using_visit/Quantitative/Query.rst

```diff
@@ -425,6 +425,24 @@ XRay Image
     |                          | of the file is "ray_bounds.vtk".             |
     +------+-------------------+----------------------------------------------+
     
+    When specifying "bov" or "rawfloats" output, the value can be either 32 or 64 bit floating point values.
+    The number of bits is determined by the number of bits in the data being processed.
+
+    When specifying "bov" output, 2 files are created for each variable.
+    One contains the "intensity" and the other the "path_length".
```

Comment:
I think where we mention names of variables here or the names of files, we should be using double back-tick (thats the Sphinx way). If so, I propose one suggested change here for this one case but I think there are several other cases below to revise. OTOH, the table above has a number of cases of that too and maybe the goal is to conform to whatever that table was using.

```suggestion
    One contains the ``intensity`` and the other the ``path_length``.
```

Reply:
All references to variables and filenames in the XRayImage section were changed.